### PR TITLE
[LC-513] add snapcraft

### PR DESCRIPTION
**Pull request by yakkle**
- add configurations, scripts, docs for citizen
- update setup.py
- add Dockerfile and README.md for snap build

### CODE REVIEW

**Review by windies21 — APPROVED** [review]

**Comment by metalg0su** [comment]
`redirectProtocol` option should be removed if following ticket is merged: https://github.com/icon-project/icon-rpc-server/pull/105

File: citizen/conf/iconrpcserver_mainnet.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        }
+    },
+    "channel": "icon_dex",
+    "port": 9000,
+    "amqpTarget": "127.0.0.1",
+    "amqpKey": "7100",
+    "gunicornConfig": {
+        "workers": 1
+    },
+    "redirectProtocol": "https"
```
Comment (metalg0su) [inline citizen/conf/iconrpcserver_mainnet.json]: `redirectProtocol` option should be removed if following ticket is merged: https://github.com/icon-project/icon-rpc-server/pull/105

File: citizen/conf/iconrpcserver_testnet.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        }
+    },
+    "channel": "icon_dex",
+    "port": 9000,
+    "amqpTarget": "127.0.0.1",
+    "amqpKey": "7100",
+    "gunicornConfig": {
+        "workers": 1
+    },
+    "redirectProtocol": "https"
```
Comment (metalg0su) [inline citizen/conf/iconrpcserver_testnet.json]: https://github.com/icon-project/loopchain/pull/448/files#r362685034

File: citizen/quick_start_ICLI.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+[ ICON CLI tool 설치 및 사용법 ]
```
Comment (metalg0su) [inline citizen/quick_start_ICLI.md]: Need to consider this document translated
(project heath check also needed: https://github.com/icon-project/icon_cli_tool)

File: citizen/quick_start_centos.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+ $ wget https://dl.bintray.com/rabbitmq/all/rabbitmq-server/3.7.7/rabbitmq-server-3.7.7-1.el7.noarch.rpm
+ $ sudo yum install rabbitmq-server-3.7.7-1.el7.noarch.rpm -y
+ ```
+
+ * Run RabbitMQ
+ ```
+ $ sudo systemctl enable rabbitmq-server
+ $ sudo systemctl start rabbitmq-server
+ ```
+
```
Comment (metalg0su) [inline citizen/quick_start_centos.md]: I think it is better to provide RabbitMQ installation guide as what Loopchain `README.md` is doing:
https://github.com/icon-project/loopchain#requirements

File: citizen/quick_start_ICLI.md (hunk 3 above)
Comment (yakkle) [reply citizen/quick_start_ICLI.md]: It will be removed.

File: citizen/conf/iconrpcserver_mainnet.json (hunk 1 above)
Comment (yakkle) [reply citizen/conf/iconrpcserver_mainnet.json]: ok. remind me.

File: citizen/conf/iconrpcserver_testnet.json (hunk 2 above)
Comment (yakkle) [reply citizen/conf/iconrpcserver_testnet.json]: ditto

File: citizen/quick_start_centos.md (hunk 4 above)
Comment (yakkle) [reply citizen/quick_start_centos.md]: done

**Review by windies21 — APPROVED** [review]

**Review by metalg0su — APPROVED** [review]

**Pull request merged**